Some types do not resolve correctly

Really cool engine! I love the simplicity of the API.

When using the template, some types seem to not be loaded through the declarations, running tsc on the `index.d.ts` declaration gives a few errors,
```sh
>  tsc types/index.d.ts
types/index.d.ts:5:16 - error TS2304: Cannot find name 'LitecanvasOptions'.

5     settings?: LitecanvasOptions,
                 ~~~~~~~~~~~~~~~~~

types/index.d.ts:6:4 - error TS2304: Cannot find name 'LitecanvasInstance'.

6 ): LitecanvasInstance
     ~~~~~~~~~~~~~~~~~~

types/index.d.ts:9:36 - error TS2304: Cannot find name 'LitecanvasOptions'.

9     function litecanvas(settings?: LitecanvasOptions): LitecanvasInstance
                                     ~~~~~~~~~~~~~~~~~

types/index.d.ts:9:56 - error TS2304: Cannot find name 'LitecanvasInstance'.

9     function litecanvas(settings?: LitecanvasOptions): LitecanvasInstance
                                                         ~~~~~~~~~~~~~~~~~~

types/index.d.ts:419:26 - error TS2304: Cannot find name 'drawCallback'.

419         draw: string[] | drawCallback,
                             ~~~~~~~~~~~~

types/index.d.ts:622:28 - error TS2304: Cannot find name 'pluginCallback'.

622     function use(callback: pluginCallback): void
                               ~~~~~~~~~~~~~~


Found 6 errors in the same file, starting at: types/index.d.ts:5

```

Perhaps they need to be explicitly imported from `types.d.ts`?

--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -1,3 +1,5 @@
+import './types'
+
 /**
  * The litecanvas constructor
  */

